dominion/dominion_ui/src/components/Search/SearchResults.js

```diff
@@ -66,16 +69,13 @@ function SearchResults({ classes, results }) {
               component="a"
               href={
                 result.type.toLowerCase() === 'country'
-                  ? `/${slugify(result.name)}`
-                  : '/'
+                  ? `/${result.slug}`
+                  : `/profiles/${result.codes[codeType]}`
               }
             >
               <Grid container direction="row" alignItems="baseline">
-                <p className={classes.level}>
-                  {result.type[0].toUpperCase() +
-                    result.type.slice(1).toLowerCase()}
-                </p>
-                <p className={classes.name}>{result.name}</p>
+                <p className={classes.level}>{Capitalize(result.type)}</p>
+                <p className={classes.name}>{Capitalize(result.name)}</p>
```

Comment:
We should try to stick to Material UI Typography as much as we can so that we get consistent fonts, spacing, etc.
```jsx
<Typography className={classes.level}  .... component="p">
```